### ISSUE TITLE
Print usage hint on `docker-machine env`

### ISSUE THREAD

**Issue by aanand** [docker/machine#877]
In light of confusing experiences like #851 (which I just experienced myself), it might be nice to show a usage hint in the output of `docker-machine env` - commented out, of course:

```
$ docker-machine env

export DOCKER_TLS_VERIFY=1
export DOCKER_CERT_PATH="/Users/aanand/.docker/machine/machines/dev"
export DOCKER_HOST=tcp://192.168.99.101:2376

# Run this command to configure your shell: eval $(docker-machine env)
```

(It needs to come afterwards because of shell expansion - if the hash is at the beginning, the whole thing will be treated as a comment :smirk:)

**Comment by mattreyuk** [comment]
#dibs

**Comment by ehazlett** [comment]
@aanand +1

**Comment by hairyhenderson** [comment]
This was fixed by #882
